stories/core/Table.stories.tsx

```diff
@@ -718,3 +759,119 @@ export const NoData: Story<TableProps> = ({ columns, data, ...rest }) => {
 NoData.args = {
   data: [],
 };
+
+export const ControlledState: Story<TableProps> = (args) => {
+  const { columns, data, ...rest } = args;
+
+  const [selectedRows, setSelectedRows] = useState<Record<string, boolean>>({});
```

Comment:
For even better example, I would say initialize with one row already selected and passed to `initialState`

Reply:
I wanted to do that initialy however `selectedRows` overrides it and I don't want people to think this is how you pass initial state. Maybe we need another story or add initial state to selection story.